apps/discord-bot/src/commands/chat/ask-command.ts

```diff
@@ -127,16 +125,17 @@ async function* getStreamedResponse(
     return
   }
 
-  question = replaceDiscordMentions(question)
-
   try {
     const response = await fetch(AGENT_STREAM_URL, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
       },
       body: JSON.stringify({
-        messages: [{ role: 'user', content: question }],
+        messages: [
+          { role: 'user', content: `I am <@!${discordUserId}>` },
```

Comment:
Is this line for testing?

Reply:
It is a refactor to remove the `I am` insertion to the prompt question.